blade-graphics/src/gles/mod.rs

```diff
@@ -100,6 +101,8 @@ struct PipelineInner {
     program: glow::Program,
     group_mappings: Box<[ShaderDataMapping]>,
     vertex_attribute_infos: Box<[VertexAttributeInfo]>,
+    blend_states: Box<[Option<crate::BlendState>]>,
+    write_masks: Box<[crate::ColorWrites]>,
```

Comment:
These two always go together in the pipeline up to the point of sending commands to GL. I.e. they are assigned together, they have the same number of elements, and they are processed together.
Can we just merge them into a single array here? We'd keep `enum Command` as you wrote it, but at the pipeline level I think these should be merged

Reply:
These boxes do not have the same size if one parameter is the same for all draw buffers and the other is not. Then, the commands also are emitted differently, one simple call for one parameter and multiple PerDrawBuffer-commands for the other parameter.

Reply:
Yes, but we don't really benefit from this path ever going to be different between the blend states and write masks. We are only doing this for compatibility reasons (not performance or anything). And for compatibility it's both or nothing.